.env

```
@@ -41,3 +41,5 @@ NEXT_PUBLIC_API_BASE="https://flower.elevateegy.com/api/v1"
 NEXTAUTHURL = "http://localhost:3000"
 NEXTAUTH_SECRET="FDZaz8bV6y8V9KTiJqjTxjs5Npvvm39hi+qmQZqK8fw="
 RESET_PASSWORD_URL="auth/resetPassword"
+NEXT_PUBLIC_GOOGLE_MAPS_API_KEY="AIzaSyBQ6_xfncKSA01yJAuNKOaPxwL8fCGjCok"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)